patches/chrome-browser-ui-views-passwords-password_save_update_view.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/views/passwords/password_save_update_view.cc b/chrome/browser/ui/views/passwords/password_save_update_view.cc
+index 9c02474ee0da521dcc35d36b957b6097ce6204d8..42b168292097039e08affdcf25c72588cc3ff4aa 100644
+--- a/chrome/browser/ui/views/passwords/password_save_update_view.cc
++++ b/chrome/browser/ui/views/passwords/password_save_update_view.cc
+@@ -416,6 +416,7 @@ void PasswordSaveUpdateView::AddedToWidget() {
+ 
+ void PasswordSaveUpdateView::OnThemeChanged() {
+   PasswordBubbleViewBase::OnThemeChanged();
++  BRAVE_PASSWORD_SAVE_UPDATE_VIEW_ON_THEME_CHANGED
```

Comment:
can't we just override the two images IDR_SAVE_PASSWORD_DARK and IDR_SAVE_PASSWORD to be empty images?

Reply:
meaning do a chromium_src override to turn them into some brave resource that is a 1x1 image or something? Also I think it will work if the image is null so maybe try overriding to be `IDR_SAVE_PASSWORD_DARK = -1` or something?